tests/e2e/must-gather_suite_test.go

```diff
@@ -156,11 +85,8 @@ var _ = Describe("Must-gather backup restore tests", func() {
 	}
 
 	DescribeTable("backup, restore applications, and must gather",
+		// TODO this function has a lot of duplications with tests/e2e/backup_restore_suite_test.go
```

Comment:
what is the goal of this test? It seems a duplication of bacjup/restore test

Reply:
This test is a duplicate of backup/restore test but only one case is included with the addition of running must gather after the test to check expected dirs and files from must-gather exists.

```go
if len(brCase.MustGatherFiles) > 0 && clusterDir != "" {
				for _, file := range brCase.MustGatherFiles {
					_, err := os.Stat(clusterDir + "/" + file)
					Expect(err).ToNot(HaveOccurred())
				}
			}
```

Reply:
Can we run it as separate test after that specific test?